### PR TITLE
Kernel: Various fixes on aarch64 to be able to use dbgln()

### PR DESCRIPTION
**Pull request by FireFox317**
This series of commits adds the ability to use dbgln() in the aarch64 build of the Kernel!

It's been a long time since I contributed, so any review is welcome!

### CODE REVIEW

**Comment by nico** [comment]
Looks good to me, fwiw.

File: Kernel/Arch/aarch64/kprintf.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#include <Kernel/kstdio.h>
+
+// FIXME: Merge the code in this file with Kernel/kprintf.cpp once the proper abstractions are in place.
+
+void kernelputstr(char const* characters, size_t length)
+{
+    if (!characters)
+        return;
+
+    auto uart = Prekernel::UART::the();
```
Comment (linusg) [inline Kernel/Arch/aarch64/kprintf.cpp:17]:
```suggestion
    auto& uart = Prekernel::UART::the();
```

Same below.

**Review by linusg — CHANGES_REQUESTED** [review]

File: Kernel/Arch/aarch64/kprintf.cpp (hunk 1 above)
Comment (FireFox317) [reply Kernel/Arch/aarch64/kprintf.cpp]: Good catch!

**Review by linusg — APPROVED** [review]

**Comment by FireFox317** [comment]
Review comments have been addressed!

**Pull request merged**